src/SFML/Graphics/RenderTexture.cpp

```diff
@@ -29,23 +29,17 @@
 #include <SFML/Graphics/RenderTextureImplFBO.hpp>
 #include <SFML/Graphics/RenderTextureImplDefault.hpp>
 #include <SFML/System/Err.hpp>
+#include <memory>
 
 
 namespace sf
 {
 ////////////////////////////////////////////////////////////
-RenderTexture::RenderTexture() :
-m_impl(nullptr)
-{
-
-}
+RenderTexture::RenderTexture() = default;
```

Comment:
How about moving `= default` to the class itself? You're losing the benefits of e.g. trivial construction doing it that way.

https://godbolt.org/z/EY6bbMTMM

Reply:
I don't think it is a worthwhile change, the class is quite big anyway and trivial construction wouldn't bring any practical benefit unless I am missing something.

Reply:
Declaring `= delete` in the header is however what we did during https://github.com/SFML/SFML/pull/1901.
It could make sense to do the same here with `= default`. A compiler-generated constructor would behave the same.

Reply:
Having `RenderTexture() = default` in the header fails to compile (at least for me with GCC 9.3).